### ISSUE TITLE
[NXT] .nxt domains fail to resolve

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/dnschain#118]
@toenu23: Following your instructions from PR #114 I am getting:

```
2015-02-16T22:33:45.244Z - error: [DNS] nxt failed to resolve (dns.coffee:148)
{ err: 'Cannot read property \'A\' of undefined',
  result: undefined,
  q: { name: 'test4.nxt', type: 1, class: 1 } }
2015-02-16T22:33:57.553Z - error: [DNS] nxt failed to resolve (dns.coffee:148)
{ err: 'Cannot read property \'A\' of undefined',
  result: undefined,
  q: { name: 'test5.nxt', type: 1, class: 1 } }
```

This is on the `mocha` branch btw, being run with:

`./bin/dnschain --nxt:connect 69.163.40.132 --nxt:port 7876`

**Comment by taoeffect** [comment]
Forgot to mention, I am resolving these like so:

- `dig @localhost -p 5333 test4.nxt`
- `dig @localhost -p 5333 test5.nxt`

**Comment by toenu23** [comment]
the reason for this is that the import of the dnsHandler function from the namecoin file isn't working. i made PR https://github.com/okTurtles/dnschain/pull/119 so the originating error isn't caught and falsely reported as nxt-error. (which imo it shouldn't)
I couldn't find a decent way to import the function, maybe it could be fully defined in blockchains.coffee, to be overridden only when necessary?

**Comment by taoeffect** [comment]
OK, so this is now fixed in 758b2fa (pushed to `mocha` just now). I also made some changes to `nxt.coffee` to get `curl -H "Host: nxt.dns" 127.0.0.1:8088/test4` to work. Please have a look at what changed there. cc @toenu23
